pkg/autoscaler/metrics/stats_scraper.go

```diff
@@ -55,6 +60,15 @@ var (
 	// ErrDidNotReceiveStat specifies the error returned by scraper when it does not receive
 	// stat from an unscraped pod
 	ErrDidNotReceiveStat = errors.New("did not receive stat from an unscraped pod")
+
+	scrapeTimeDistribution = view.Distribution(
```

Comment:
we already have a standard distribution in pkg/metrics?

Reply:
I missed that. I'll use it instead.